test/unit/attributes/attribute_test.rb

```diff
@@ -225,4 +225,16 @@
       attribute.send(:valid_numeric?, '1/2').must_equal false
     end
   end
+
+  describe 'to_ruby method' do
+    it 'generates the code for the attribute' do
+      attribute = Inspec::Attribute.new('application_port', description: 'The port my application uses', value: 80)
+      attribute.to_ruby.must_equal <<-RUBY.chomp
+attr_application_port = attribute('application_port',{
+  value: 80,
+  description: 'The port my application uses',
+})
+RUBY
+    end
```

Comment:
I think we might also want to try to eval the code.  We care that we not only generate lines with certain things in it, but that code be valid code.

Reply:
That sounds good. It looks like `eval` always returns `nil` and all I can test is that nothing raises.
It also looks like MiniTest has no assertion for "must_not_raise" and all I can do is call eval() bare and let its unhandled exception fail the test?

Let me know if you had a better approach in mind.

Reply:
Yes, that's how it works - nothing is raised; and then I usually put a test afterwards, to ensure it worked.

Reply:
Awesome! Thank you!